integration-tests/docker/environment-configs/historical-for-query-error-test

```diff
@@ -29,4 +30,4 @@ druid_processing_numThreads=2
 druid_query_groupBy_maxOnDiskStorage=300000000
 druid_segmentCache_locations=[{"path":"/shared/druid/indexCache-query-error-test","maxSize":5000000000}]
 druid_auth_basic_common_cacheDirectory=/tmp/authCache/historical-query-error-test
-druid_server_https_crlPath=/tls/revocations.crl
+druid_server_https_crlPath=/tls/revocations.crl
```

Comment:
extra change?

Reply:
I rolled back the file, not sure why is it highlighting